.github/workflows/test-external.yml

```diff
@@ -72,14 +72,6 @@ jobs:
             profile: "halmos"
 
     steps:
-      # TODO: remove this step once halmos-builder package is public
-      - name: Login to GitHub Container Registry
-        uses: docker/login-action@v3
-        with:
-          registry: ghcr.io
-          username: ${{ github.actor }}
-          password: ${{ secrets.GITHUB_TOKEN }}
-
```

Comment:
are we going to make halmos-builder public as well? if i remember correctly, docker build previously failed without this login credential.

Reply:
ah that's right, I forgot about that. Let's do it!